static/js/dialog.js

```diff
@@ -203,17 +217,16 @@ dialog.prototype._focusOnContainer = function() {
   // fix for iOS: when opening the dialog, we need to force focus on padOuter
```

Comment:
a gente ainda precisa desse fix no iOS?

Reply:
Não sei, não testei no iOS depois disso. Mas resolvi deixar pq quando (se) no futuro a gente for fazer os ajustes para isso, ao menos esse a gente não precisaria re-fazer.